src/main/scala/cognite/spark/SdkV1Rdd.scala

```diff
@@ -52,10 +57,11 @@ case class SdkV1Rdd[A](
       .next
       .reduce(_.merge(_))
 
-    val queue: ArrayBlockingQueue[Vector[A]] =
-      new ArrayBlockingQueue[Vector[A]](config.parallelismPerPartition * 2)
+    val queue =
+      new EitherQueue(config.parallelismPerPartition * 2)
```

Comment:
Why "* 2"?

Reply:
"Seems to work" is the answer here, I guess. The queue never fills up, during testing at least, as Spark drains the queue quite a lot faster than the API delivers new data. I figured `*2` was an all right setting here, in the worst case it would store 2 chunks per stream before blocking. If the queue is getting filled up we might as well start blocking, but at least for the test setup we get the same performance by just using a constant value like 5.